src/gui/search/searchsortmodel.h

```diff
@@ -30,10 +30,12 @@
 #define SEARCHSORTMODEL_H
 
 #include <QSortFilterProxyModel>
+#include <QStringList>
 #include "base/utils/string.h"
 
 class SearchSortModel: public QSortFilterProxyModel
 {
+    using base = QSortFilterProxyModel;
```

Comment:
Forgot to say earlier... We start the names of complex types with a capital letter. The same applies to the aliases (etc.) of such types. So there must be `Base` (although I don't see much point to use this alias).